src/strings-in-block/class-composite.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace WPML\PB\Gutenberg\StringsInBlock;
+
+class Composite implements StringsInBlock {
```

Comment:
@strategio I don't get the meaning of `Composite`

Reply:
It's the [composite pattern](https://sourcemaking.com/design_patterns/composite) @andreasciamanna. I don't have a better idea to name this...

Reply:
Unless it represents an entity, I usually prefer using verbs or adjectives.

What about "Compose"?

That's the purpose of the composite pattern after all: to **compose** different entities into something which can be handled as a whole.

Reply:
Done!